src/components/formInput/view/formInputView.js

```diff
@@ -13,123 +13,107 @@ import { getResizedAvatar } from '../../../utils/image';
 // Styles
 import styles from './formInputStyles';
 
-class FormInputView extends Component {
-  /* Props
-   * ------------------------------------------------
-   *   @prop { string }     placeholder       - Place holder text.
-   *   @prop { string }     type              - Input type.
-   *   @prop { boolean }    isFirstImage      - Render image from steem.
-   *   @prop { boolean }    isEditable        - Can permission edit.
-   *   @prop { boolean }    isValid           - This delegate input valit or not.
-   *   @prop { boolean }    secureTextEntry   - For hiding password value.
-   */
-  constructor(props) {
-    super(props);
+const FormInputView = ({
+  placeholder,
+  type,
+  isFirstImage,
+  isEditable,
+  leftIconName,
+  rightIconName,
+  secureTextEntry,
+  iconType,
+  wrapperStyle,
+  height,
+  inputStyle,
+  onChange,
+  isValid,
+  value,
+}) => {
+  const [_value, setValue] = useState(value || '');
+  const [inputBorderColor, setInputBorderColor] = useState('#e7e7e7');
+  const [_isValid, setIsValid] = useState(true);
 
-    this.state = {
-      value: props.value || '',
-      inputBorderColor: '#e7e7e7',
-      isValid: true,
-    };
-  }
+  const _handleOnChange = text => {
+    setValue(text);
 
-  // Component Functions
-  _handleOnChange = value => {
-    const { onChange } = this.props;
-
-    this.setState({ value });
     if (onChange) {
-      onChange(value);
+      onChange(text);
     }
   };
 
-  _handleOnFocus = () => {
-    this.setState({ inputBorderColor: '#357ce6' });
+  const _handleOnFocus = () => {
+    setInputBorderColor('#357ce6');
   };
 
-  // Component Life Cycles
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const { isValid } = this.props;
-
-    if (nextProps.isValid !== isValid) {
-      this.setState({ isValid: nextProps.isValid });
-    }
-  }
+  // Workaround for android context (copy/paste) menu
+  useEffect(() => {
+    setValue(' ');
```

Comment:
why whitespace?

Reply:
we should update to input value for resolve to context menu issue, this issue can due from react-navigation but i couldnt find any solution for that

Reply:
Yes, I also checked react-navigation before, tried package upgrades, didn't help anything. Something new or the way we are using it might be buggy. One more theory I have is that, it might be Modal component related.

`This is what I see on terminal`

  WARN  Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.
                                
 * Move data fetching code or side effects to componentDidUpdate.
 * If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
 * Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.
                                
 Please update the following components: **ModalDropdown**, Popover

Reply:
this warn mostly related with 3th party libraries, we changed all componentWillReceiveProps function with UNSAFE_componentWillReceiveProps and we are refactoring with hooks. But some libraries didnt change yet

Reply:
Yes, read my comment on Discord. I have updated couple components locally so we don't have to wait some packages (some of them inactive). So we can fork, make change ourselves.